examples/rust/fft/README.md

```diff
@@ -0,0 +1,95 @@
+# Fast Fourier Transform in SingleStoreDB
+
+## Introduction
+
+[Fast Fourier transform](https://en.wikipedia.org/wiki/Fast_Fourier_transform) is an algorithm that computes the discrete Fourier transform (DFT) of a sequence, or its inverse (IDFT).
+Fourier analysis converts a signal from its original domain (often time or space) to a representation in the frequency domain and vice versa. The DFT is obtained by decomposing a sequence of values into components of different frequencies.
+Fast Fourier transforms are widely used for applications in engineering, music, science, and mathematics. The FFT is used in digital recording, sampling, additive synthesis and pitch correction software.
+
+This library uses [rustfft](https://docs.rs/rustfft/latest/rustfft/) for algorithm implementation. This is a high-performance, SIMD-accelerated FFT library, compute FFTs of any size, in O(nlogn) time. Support for hardware acceleration (Avx, Neon, Sse) from rustfft is not ported to this module.
+
+## Contents
+This library provides the following database objects.
+
+### `st_planner_forward(len: u8, buffer: [Complex<f64>])`
```

Comment:
Let's use the database types here.  So, for this case it would be:
```sql
st_planner_forward(len TINYINT UNSIGNED NOT NULL, buffer RECORD(re DOUBLE NOT NULL, im DOUBLE NOT NULL)
```
This is a bit long for the heading line, so maybe just put `st_planner_forward` in the heading and then put the UDF signature in the description.